spec/models/user_spec.rb

```diff
@@ -49,4 +49,20 @@
       expect(user.admin?).to be_truthy
     end
   end
+
+  describe "reassign_topics_to_new_users" do
+    let(:other_user) { create(:user) }
+    let(:my_topic) { create(:topic, user: user) }
+    let(:other_user_bookmark) { create(:bookmark, topic: my_topic, user: other_user) }
+
+    it "should reassign a topic to other_user" do
+      delete :destroy, {id: user.id}
+      expect(assigns(:user)).to eq(other_user)
+    end
+    it "should delete a topic if there is no user to reassign it to" do
+      delete :destroy, {id: user.id}
+      count = User.where({id: user.id}).size
```

Comment:
oh, and lastly, the description for this test doesn't match the code that is actually being tested.  it sounds like you want to test that a topic gets deleted, but the code looks like a user is getting deleted and not checking topics at all.